mdast_util_to_markdown/src/configure.rs

```diff
@@ -65,6 +65,10 @@ pub struct Options {
     /// Setext headings cannot be used for empty headings or headings with a
     /// rank of three or more.
     pub setext: bool,
+    /// (`bool`, default: `true`) — whether to support math (text) with a single dollar.
+    /// Single dollars work in Pandoc and many other places, but often interfere with “normal”
+    /// dollars in text. If you turn this off, you can still use two or more dollars for text math
```

Comment:
```suggestion
    /// dollars in text.
    /// If you turn this off, you can still use two or more dollars for text math
```

And then I try to break after sentences.
the reasoning for that is roughly https://sembr.org: it improves diffing and reasoning.

Reply:
Oh IC, I usually use a line in my editor to indicate when to break, this is interesting though thanks for letting me know.